algorithms/maths/recursive_binomial_coefficient.py

```diff
@@ -0,0 +1,31 @@
+def recursive_binomial_coefficient(n,k):
+    """Calculates the binomial coefficient, C(n,k), with n>=k using recursion
+    Time complexity is O(k), so can calculate fairly quickly for large values of k.
+
+    >>> recursive_binomial_coefficient(5,0)
+    1
+
+    >>> recursive_binomial_coefficient(8,2)
+    28
+
+    >>> recursive_binomial_coefficient(500,300)
+    5054949849935535817667719165973249533761635252733275327088189563256013971725761702359997954491403585396607971745777019273390505201262259748208640
+
+    >>> recursive_binomial_coefficient(4,5)
+    'Invalid Inputs'
+
+    """
+
+    if k>n:
+        return 'Invalid Inputs'
```

Comment:
it is better to raise error this case.
if you return two types (string, num) consumers need to branch out by the response type

Reply:
fixed 👍